test/gui/shared/steps/steps.py

```diff
@@ -239,6 +239,21 @@ def step(context, receiver, resource, permissions):
     openSharingDialog(context, resource)
     shareItem = SharingDialog()
     shareItem.addCollaborator(receiver, permissions)
+    clickButton(waitForObject(shareItem.SHARING_DIALOG_CLOSE_BUTTON))
```

Comment:
we can create a function `closeSharingDialog`  in pageObject and call it here and other places when needed.